### PR TITLE
#157731609 Present deactivated users 

### PR DESCRIPTION
**Pull request by King-Benx**
### What does this PR do?
Presents a way of showing both active and deactivated users
#### Description of Task to be completed?
Previously, wger only presented a list of all users within a gym and this was inefficient because it did not provide a way of summarising the users based on their inactivity or activity within a gym.

This PR solves this issue by providing a tab that can be switched to either show active or inactive users within the gym.
#### Screenshots
active_users

Display of active users of a gym
inactive_users

Display of inactive users of a gym
#### How should this be manually tested?
1. Follow the readme.rst procedures on how to set up the application locally
2. Login into the system using the default administrator credentials: username (admin), password (admin)
3. In the navigational bar select the profile link at the far right to display the administration list.
4. Select either gyms or default gyms.
5. By default active users are being shown, to display deactivated users, select the Inactive tab button

#157731609

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (tibetegya) [inline .vscode/settings.json]: remove .vscode/ folder

File: wger/core/templates/mobile/template.html [hunk 2]
```diff
@@ -56,7 +56,7 @@
     <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap-custom.css' %}">
     <link rel="stylesheet" type="text/css" href="{% static 'css/workout-manager-common.css' %}">
     <link rel="stylesheet" type="text/css" href="{% static 'css/workout-manager-mobile.css' %}">
-    <link rel="stylesheet" type="text/css" href="{% static 'bower_components/shariff/build/shariff.min.css' %}">
```
Comment (tibetegya) [inline wger/core/templates/mobile/template.html]: do not change the build folder for shariff static files

File: wger/settings_global.py [hunk 3]
```diff
@@ -91,12 +91,12 @@
 # added list of external libraries to be installed by bower
 BOWER_INSTALLED_APPS = (
     'jquery#2.1.x',
-    'bootstrap',
+    'bootstrap#3.3.5',
     'd3',
     'shariff',
```
Comment (tibetegya) [inline wger/settings_global.py]: use shariff#1.24.1 which is the one that has build folder

File: wger/settings_global.py [hunk 4]
```diff
@@ -91,12 +91,12 @@
 # added list of external libraries to be installed by bower
 BOWER_INSTALLED_APPS = (
     'jquery#2.1.x',
-    'bootstrap',
+    'bootstrap#3.3.5',
```
Comment (tibetegya) [inline wger/settings_global.py]: just use bootstrap#3 for uniformity

File: wger/utils/language.py [hunk 5]
```diff
@@ -52,7 +52,7 @@ def load_language(language_code=None):
         language = Language.objects.get(short_name=used_language)
     except ObjectDoesNotExist:
         # No luck, load english as our fall-back language
-        language = Language.objects.get(short_name="en")
+        language = Language.objects.get_or_create(short_name="en")
```
Comment (tibetegya) [inline wger/utils/language.py]: change this line back to `language = Language.objects.get(short_name="en")`

File: .vscode/settings.json (hunk 1 above)
Comment (King-Benx) [reply .vscode/settings.json]: I will do that on my next commit

File: wger/core/templates/mobile/template.html (hunk 2 above)
Comment (King-Benx) [reply wger/core/templates/mobile/template.html]: I will do that on my next commit

File: wger/settings_global.py (hunk 3 above)
Comment (King-Benx) [reply wger/settings_global.py]: Okay thank you, will work on that

File: wger/settings_global.py (hunk 4 above)
Comment (King-Benx) [reply wger/settings_global.py]: I will do that on my next commit

File: wger/utils/language.py (hunk 5 above)
Comment (King-Benx) [reply wger/utils/language.py]: I will do that on my next commit

**Comment by emugaya** [comment]
@King-Benx Rebase..

**Comment by King-Benx** [comment]
Thanks for the feedback, I am going to rebase in order to resolve the conflicts

File: .gitignore [hunk 6]
```diff
@@ -62,6 +62,10 @@ target/
 # External Libraries 
 # wger/core/static/bower_components
 node_modules
-*.DS_Store
-.vscode/
-package-lock.json
+
+# Virtual Environment
+venv-django/
+<<<<<<< HEAD
```
Comment (emugaya) [inline .gitignore]: What is this doing here?

**Comment by emugaya** [comment]
@King-Benx Why are you pushing`venv-django` virtual env folder.

**Comment by King-Benx** [comment]
I have resolved all issues that were raised

**Pull request merged**